### DISCUSSION TITLE
Catalog item vs model card

### DISCUSSION THREAD

**Discussion by binkymilk** [thinkingmachines/unicef-ai4d-research-bank#46] [General]
Hi @AnthonyMockler! Moving the discussion here for this specific feedback from Figma:

> It's clear how accurate this data is (When compared against another dataset, or the validation / holdout dataset)
Evaluation metric should probably link to a model card? Or are we going to try and pack it all in on this? Model card example: https://modelcards.withgoogle.com/object-detection 

I'm curious about what your idea of a catalog item is and how it relates to a model card.

Right now, my idea of a catalog item is that it can be either a dataset or a model. They essentially will have the same layout in the web app, but some fields may be different. An example of this would be, a model will have a list of training datasets, while a dataset (if I understood it correctly) will have none.

Let me know your thoughts! 🙏  This will also clear up the confusion about having two different layouts for data and model. Thanks!

Figma comment: https://www.figma.com/file/kh54B7nOM3PeE0OqnpqfU3?node-id=15:456#387336022
Github issue: https://github.com/thinkingmachines/unicef-ai4d-research-bank/issues/2

**Comment by ghost** [comment]
Hi @AnthonyMockler , We'd love to hear your thoughts on this so we can integrate your feedback as soon as possible!

**Reply by AnthonyMockler** [reply]
@binkymilk A model card is something Google are trying to make a thing - It's a standardised, structured way of capturing some of the important characteristics of a particular model:
Long research paper here: https://arxiv.org/abs/1810.03993
An example of one here: https://modelcards.withgoogle.com/object-detection
More approachable example: https://github.com/openai/gpt-3/blob/master/model-card.md

We *could* adopt the model card structure for the 'results' tab? And get contributors to include a markdown formatted model card? Open to ideas
